web/src/pages/Plugin/PluginMarket.tsx

```diff
@@ -20,6 +20,7 @@ import { PageHeaderWrapper } from '@ant-design/pro-layout';
 
 import PluginPage from '@/components/Plugin';
 import { fetchList, createOrUpdate } from './service';
+import { formatMessage } from 'umi';
```

Comment:
ok, we need to use

```suggestion
import { useIntl } from 'umi';
```

here, and use `const { formatMessage } = useIntl()`